### ISSUE TITLE
webout macro not found

### ISSUE THREAD

**Issue by argenx-pmason** [sasjs/server#319]
When I run code from vs code and I have webout macros, then it doesnt find the macro. I put the macro into a location but then it found an old version of ms_webout which was missing the max parameter. So there's some things wrong in there that need fixing please.

**Comment by allanbowe** [comment]
Hi @argenx-pmason - we looked into this and our expectation is that the old version of `ms_webout()` is in relation to a previous install.

https://github.com/sasjs/server/issues/318

A few versions ago (when we enabled a shared DRIVE_LOCATION across multiple SASjs server instances) we moved the sasautos to $(DRIVE_LOCATION)/sas/sasautos.  By default, DRIVE_LOCATION is $(SASJS_ROOT)/drive.

To fix this up I suggest simply removing the old sasjs_root/sasautos folder.

The issue above will also mean that the SASjs Server version of `webout()` will be auto-added to sasautos on first install.
